docker/rskj/node.conf

```diff
@@ -81,6 +81,7 @@ blockchain.config {
     },
     consensusRules = {
         rskip97 = -1 # disable orchid difficulty drop
+        rskip252 = -1
```

Comment:
Describe in comment (as done above) and perhaps add link https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP252.md

Reply:
https://medium.com/iovlabs-innovation-stories/more-on-the-security-against-fee-sniping-in-rsk-602a59591d7f
just to make sure that the commit reader understands potential implications?